nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -1145,11 +1145,29 @@ systemd.services.nginx.serviceConfig.User = lib.mkForce "root";
      As well as this, the options <literal>security.acme.acceptTerms</literal> and either
      <literal>security.acme.email</literal> or <literal>security.acme.certs.&lt;name&gt;.email</literal>
      must be set in order to use the ACME module.
-     Certificates will be regenerated on activation, no account or certificate will be migrated from simp-le.
-     In particular private keys will not be preserved. However, the credentials for simp-le are preserved and
-     thus it is possible to roll back to previous versions without breaking certificate generation.
-     Note also that in contrary to simp-le a new private key is recreated at each renewal by default, which can
-     have consequences if you embed your public key in apps.
+    </para>
+    <para>Please note:
+     <itemizedlist>
+      <listitem><para>Certificates will be regenerated on activation, no account or certificate will be migrated from simp-le.
+        In particular private keys will not be preserved. However, the credentials for simp-le are preserved and
+        thus it is possible to roll back to previous versions without breaking certificate generation.
+        Note also that in contrary to simp-le a new private key is recreated at each renewal by default, which can
+        have consequences if you embed your public key in apps.
+       </para>
+       <para>
+        A separate Let's-Encyrpt-account is registered for each distinctive
```

Comment:
```suggestion
        A separate Let's-Encrypt-account is registered for each distinctive
```

Reply:
Also, the rest of the changelog/documentation seems to be pretty `Let's-Encrypt` unspecific - the module uses the ACME protocol, and Let's-Encrypt is the default provider. I'd propose to reword this to match this style.